GhostNetwork.Gateway.Api/Users/SecuritySettingsController.cs

```diff
@@ -0,0 +1,49 @@
+﻿using GhostNetwork.Gateway.Users;
+using GhostNetwork.Profiles.Model;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Threading.Tasks;
+
+namespace GhostNetwork.Gateway.Api.Users
+{
+    [Route("[controller]")]
+    [ApiController]
+    public class SecuritySettingsController : ControllerBase
+    {
+        private readonly IUsersStorage usersStorage;
+
+        public SecuritySettingsController(IUsersStorage usersStorage)
+        {
+            this.usersStorage = usersStorage;
+        }
+
+        [HttpGet("{userId:guid}")]
```

Comment:
Dont don't we will add any functionality to view or change others' settings, so endpoints should look like
[GET] /securitysettings
[POST] /securitysettings